docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+# Kubernetes on CenturyLink Cloud
```

Comment:
File needs to be added to the left nav; can you find a good spot in /_data/guides.yml?